src/app/modules/tasks/containers/CreateTaskForm.tsx

```diff
@@ -0,0 +1,16 @@
+import { reduxForm } from 'redux-form';
+import { default as TaskForm, TaskFormProps } from '../components/TaskForm';
+import { SheriffTask, SheriffAbility } from '../../../api/index';
+import { createTask } from '../actions';
+
+
+const createSheriffFormFactory = reduxForm<TaskFormProps, any>({ 
```

Comment:
How do you feel about the more condensed form of this i.e.:

```js
const formConfig = {
  form: 'CreateTaskForm',
  onSubmit: (values,dispatch,props)=>{...}
};

export default reduxForm<TaskFormProps,any>(formConfig)(TaskForm);
```

Reply:
Updated as follows: 
```js 
export default reduxForm<TaskFormProps, any>({ 
    form: 'CreateTaskForm',
    onSubmit: (values:SheriffTask|any, dispatch, props) =>{
        let newTask = Object.assign({}, values, {requiredAbilities:SheriffAbility.All}, {sheriffIds:[]})
        dispatch(createTask(newTask));
    } 
})(TaskForm);
```

did the same update on the CreateSheriffForm